src/features/analytics_panel/components/AnalyticsDataList/AnalyticsDataList.tsx

```diff
@@ -1,50 +1,54 @@
-import { TooltipTrigger } from '~components/TooltipTrigger';
+import { nanoid } from 'nanoid';
+import { Text } from '@konturio/ui-kit';
+import { InfoErrorOutline16 } from '@konturio/default-icons';
+import { Trans } from 'react-i18next';
+import { useAtom } from '@reatom/react';
+import { analyticsResourceAtom } from '../../atoms/analyticsResource';
 import s from './AnalyticsData.module.css';
 import type { AnalyticsData } from '~core/types';
-import type { PropsWithChildren } from 'react';
 
 interface AnalyticsDataListProps {
   data?: AnalyticsData[] | null;
 }
 
-const Sub = ({ children }: PropsWithChildren) => (
-  <span style={{ fontSize: '.7em', verticalAlign: 'super' }}>{children}</span>
-);
+const formatFractionalNumbers = (num: number) =>
```

Comment:
Does this forrmatter meets the criteria:

_rounding: 2 decimal places, if the decimal part contains only zeros - don't display the decimal part_

For example we have `num = 2.004`

Then we will get `2.00`

Reply:
fixed